### PR TITLE
Allow a view to return a PDF containing a (scaled) PNG

### PR DESCRIPTION
**Pull request by RickMohr**
Install Cairo graphics library via `python-cairo` package

### CODE REVIEW

**Comment by RickMohr** [comment]
To test, hit 
`/group/<group_slug>/event/<event_slug>/printable-map/`

**Comment by RickMohr** [comment]
@azavea-bot rebuild

**Comment by jwalgran** [comment]
My provisioner failed. pypi says the latest version of ``python-cairo`` is 1.10

```
TASK: [nyc-trees.app | Install python-cairo (for PDF generation)] ************* 
failed: [app] => {"failed": true}
stderr: E: Version '1.8.8' for 'python-cairo' was not found

stdout: Reading package lists...
Building dependency tree...
Reading state information...

msg: 'apt-get install 'python-cairo=1.8.8'' failed: E: Version '1.8.8' for 'python-cairo' was not found

**Comment by RickMohr** [comment]
Sorry @jwalgran, should be fixed now. You could wait for the PR builder to be sure.

**Comment by jwalgran** [comment]
+1 I got a PDF of the logo

File: deployment/ansible/roles/nyc-trees.app/tasks/dependencies.yml [hunk 1]
```diff
@@ -13,6 +13,9 @@
 - name: Install PostgreSQL client
   apt: pkg=postgresql-client-{{ postgresql_version }}={{ postgresql_package_version }}
 
+- name: Install python-cairo (for PDF generation)
+  apt: pkg=python-cairo=1.8.8-1ubuntu5 state=present
```
Comment (hectcastro) [inline deployment/ansible/roles/nyc-trees.app/tasks/dependencies.yml]: This should probably be `python-cairo=1.8.8*`.

Comment (RickMohr) [inline deployment/ansible/roles/nyc-trees.app/tasks/dependencies.yml]: Thanks, that's better.

**Comment by RickMohr** [comment]
Closed in favor of #824

**Pull request closed without merging**